src/inputs/roleEditor/RoleEditor.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Grid, Typography, Avatar, makeStyles } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { CheckmarksMultipleSelect } from '../CheckmarksMultipleSelect';
+
+const useStyles = makeStyles((theme) => ({
+  avatar: {
+    backgroundColor: theme.palette.textSecondary,
+  },
+  userGroup: {
+    display: 'flex',
+    alignItems: 'center',
+    gap: '15px',
+    marginLeft: 0,
+  },
+  rolesEditor: {
+    display: 'flex',
+    alignItems: 'center',
+    marginTop: '15px',
+    marginBottom: '15px',
+  },
+}));
+export const RoleEditor = ({
+  agent,
+  agentRoles,
+  allRoles,
+  icon,
+  onRoleChecked,
+  removeAccessLabel,
+  isRemoveAccessAvailable,
+  onRemoveAccess,
+  labels,
+  isReadOnly,
+}) => {
+  const classes = useStyles();
+  return (
+    <Grid item xs={12} className={classes.rolesEditor}>
+      <Grid item xs={7} className={classes.userGroup}>
+        <Avatar className={classes.avatar}>{icon}</Avatar>
```

Comment:
Is this icon the specified user's one? If so, we should be using the UI component computing the color and initial letter we are using in the AppBar.

Reply:
At this point we don't have access to the user name yet in this component, but only to their email (which would generate a `DefaultAvatar` with a different color than they're used to, because we use the user name in the `UserInfo` component). I suggest to keep the generic Avatar for now, and modify the code later to use the `DefaultAvatar` :slightly_smiling_face: